src/components/Input/Input.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { InputWrapper } from './Input.styles';
+
+const Input = ({ type, value, onChange }) => (
```

Comment:
You actually don't need this component since you're only copying props over. Like the CTAButton

Reply:
My thought of having a component was to have PropTypes but you are right. Will remove now